lib/msf/core/modules/external/bridge.rb

```diff
@@ -197,8 +197,19 @@ def self.applies?(module_name)
 
   def initialize(module_path, framework: nil)
     super
-    gopath = ENV['GOPATH'] || ''
-    self.env = self.env.merge({ 'GOPATH' => File.expand_path('../go', __FILE__) + File::PATH_SEPARATOR + gopath})
+    default_go_path = ENV['GOPATH'] || ''
+    shared_module_lib_path = File.dirname(module_path) + "/shared"
```

Comment:
Inconsistent with the below code since it hardcodes the `/` instead of using `File::PATH_SEPARATOR`.

Consider using `File.join` for this:
```
shared_module_lib_path = File.join(File.dirname(module_path), 'shared')
```